cogs/qrchecker/qrchecker.py

```diff
@@ -145,6 +169,43 @@ async def qrchecker(self, ctx: Context):
         """Configure QR code checker"""
         pass
 
+    @qrchecker.command("show")
+    async def show(self, ctx: Context):
+        """Show current settings"""
+        pixels = await self.config.get_attr(KEY_MAX_IMAGE_DIMENSIONS)()
+        enabled = await self.config.guild(ctx.guild).get_attr(KEY_ENABLED)()
+        msg = "**__Global settings:__**\n"
+        msg += f"Max pixels: **{'Unlimited' if not pixels else pixels} pixels**\n\n"
+        msg += "**__Guild settings:__**\n"
+        msg += f"Enabled: **{'Yes' if enabled else 'No'}**"
```

Comment:
How about having variables for _max pixels_ and _enabled_ info strings?

```suggestion

        pixelsStr = f"{pixels or "Unlimited"} pixels"
        enabledStr = "Yes" if enabled else "No"

        globalSettingsStr = "\n".join((
            bold(italic("Global settings:"), escape_formatting=False),
            f"Max pixels: {bold(pixelsStr)}",
        ))
        guildSettingsStr = "\n".join((
            bold(italic("Guild settings:"), escape_formatting=False),
            f"Enabled: {bold(enabledStr)}",
        ))

        msg = "\n".join((globalSettingsStr, guildSettingsStr))
```

I don't feel too strongly about `bold` and `italic` from `redbot.core.utils.chat_formatting` because they have `escape_formatting` true by default...

Reply:
I'll skip on `chat_formatting` for now unless you strongly feel for it.

Reply:
WFM. Can do markdown literals.